Makefile

```diff
@@ -17,10 +17,10 @@ runctestimage:
 	docker build -t $(RUNC_TEST_IMAGE) -f $(TEST_DOCKERFILE) .
 
 test: runctestimage
-	docker run --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_TEST_IMAGE) make localtest
+	docker run -e TEST_ARGS=${TEST_ARGS} --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_TEST_IMAGE) make localtest
```

Comment:
Can we call it `TESTFLAGS`? We sorta used to it in docker :)
Also I think you can use just `-e TEST_ARGS` to pass this variable inside container, but I'm not sure.